src/data/java/com/github/minecraftschurlimods/arsmagicalegacy/data/AMAbilityProvider.java

```diff
@@ -17,37 +17,37 @@ class AMAbilityProvider extends AbilityProvider {
-        createAbility(AMAbilities.FROST_WALKER.get().getId(),           AMAffinities.ICE.get(),       Range.ofLowerBound(1))    .build(consumer);
-        createAbility(AMAbilities.SLOWNESS.get().getId(),               AMAffinities.ICE.get(),       Range.ofBounds(0.5, 1))   .build(consumer);
-        createAbility(AMAbilities.SPEED.get().getId(),                  AMAffinities.LIGHTNING.get(), Range.ofBounds(0.01, 1))  .build(consumer);
-        createAbility(AMAbilities.STEP_ASSIST.get().getId(),            AMAffinities.LIGHTNING.get(), Range.ofLowerBound(1))    .build(consumer);
-        createAbility(AMAbilities.WATER_DAMAGE_LIGHTNING.get().getId(), AMAffinities.LIGHTNING.get(), Range.ofBounds(0.5, 1))   .build(consumer);
-        createAbility(AMAbilities.THORNS.get().getId(),                 AMAffinities.NATURE.get(),    Range.ofBounds(0.01, 1))  .build(consumer);
-        createAbility(AMAbilities.SATURATION.get().getId(),             AMAffinities.NATURE.get(),    Range.ofLowerBound(1))    .build(consumer);
-        createAbility(AMAbilities.NETHER_DAMAGE_NATURE.get().getId(),   AMAffinities.NATURE.get(),    Range.ofBounds(0.5, 1))   .build(consumer);
-        createAbility(AMAbilities.SMITE.get().getId(),                  AMAffinities.LIFE.get(),      Range.ofBounds(0.01, 1))  .build(consumer);
-        createAbility(AMAbilities.REGENERATION.get().getId(),           AMAffinities.LIFE.get(),      Range.ofLowerBound(1))    .build(consumer);
-        createAbility(AMAbilities.NAUSEA.get().getId(),                 AMAffinities.LIFE.get(),      Range.ofBounds(0.5, 1))   .build(consumer);
-        createAbility(AMAbilities.MANA_REDUCTION.get().getId(),         AMAffinities.ARCANE.get(),    Range.ofBounds(0.01, 1))  .build(consumer);
-        createAbility(AMAbilities.CLARITY.get().getId(),                AMAffinities.ARCANE.get(),    Range.ofLowerBound(1))    .build(consumer);
-        createAbility(AMAbilities.MAGIC_DAMAGE.get().getId(),           AMAffinities.ARCANE.get(),    Range.ofBounds(0.5, 1))   .build(consumer);
-        createAbility(AMAbilities.POISON_RESISTANCE.get().getId(),      AMAffinities.ENDER.get(),     Range.ofBounds(0.5, 1))   .build(consumer);
-        createAbility(AMAbilities.NIGHT_VISION.get().getId(),           AMAffinities.ENDER.get(),     Range.ofBounds(0.5, 1))   .build(consumer);
-        createAbility(AMAbilities.ENDERMAN_PUMPKIN.get().getId(),       AMAffinities.ENDER.get(),     Range.ofLowerBound(1))    .build(consumer);
-        createAbility(AMAbilities.LIGHT_HEALTH_REDUCTION.get().getId(), AMAffinities.ENDER.get(),     Range.ofBounds(0.5, 0.99)).build(consumer);
-        createAbility(AMAbilities.WATER_HEALTH_REDUCTION.get().getId(), AMAffinities.ENDER.get(),     Range.ofBounds(0.5, 0.99)).build(consumer);
+        createAbility(AMAbilities.FIRE_RESISTANCE.get().getId(),        AMAffinities.FIRE.get(),      MinMaxBounds.Doubles.between(0.01, 1)).build(consumer);
```

Comment:
Please fix the formatting for this and the following lines (all .build() statements should start at the same column),

Reply:
the original formatting was wrong
only add space for parameters not method calls